diff --git a/browser/app/macbuild/Contents/MacOS-files.in b/browser/app/macbuild/Contents/MacOS-files.in
index 2f420f7..3551c66 100644
--- a/browser/app/macbuild/Contents/MacOS-files.in
+++ b/browser/app/macbuild/Contents/MacOS-files.in
@@ -17,7 +17,6 @@
 /crashhelper
 #endif
 /nmhproxy
-/pingsender
 /pk12util
 /ssltunnel
 /xpcshell
diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
index dbfb287..7d47b40 100644
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -397,10 +397,6 @@ bin/libfreebl_64int_3.so
 @BINPATH@/crashhelper@BIN_SUFFIX@
 #endif
 
-; [ Ping Sender ]
-;
-@BINPATH@/pingsender@BIN_SUFFIX@
-
 ; [ Native Messaging Host Proxy ]
 ;
 #if defined(XP_WIN) || defined(XP_MACOSX)
diff --git a/browser/installer/windows/nsis/shared.nsh b/browser/installer/windows/nsis/shared.nsh
index 8e5e75d..1fbc59f 100644
--- a/browser/installer/windows/nsis/shared.nsh
+++ b/browser/installer/windows/nsis/shared.nsh
@@ -1765,7 +1765,6 @@ FunctionEnd
   Push "crashreporter.exe"
   Push "default-browser-agent.exe"
   Push "nmhproxy.exe"
-  Push "pingsender.exe"
   Push "updater.exe"
   Push "mozwer.dll"
   Push "${FileMainEXE}"
