diff --git a/python/mozbuild/mozbuild/artifacts.py b/python/mozbuild/mozbuild/artifacts.py
index c4ab5ea..78100e6 100644
--- a/python/mozbuild/mozbuild/artifacts.py
+++ b/python/mozbuild/mozbuild/artifacts.py
@@ -634,7 +634,6 @@ class LinuxArtifactJob(ArtifactJob):
         "{product}/dependentlibs.list",
         "{product}/{product}",
         "{product}/{product}-bin",
-        "{product}/pingsender",
         "{product}/plugin-container",
         "{product}/updater",
         "{product}/glxtest",
@@ -732,7 +731,6 @@ class MacArtifactJob(ArtifactJob):
                 "{product}-bin",
                 "*.dylib",
                 "nmhproxy",
-                "pingsender",
                 "plugin-container.app/Contents/MacOS/plugin-container",
                 "updater.app/Contents/MacOS/org.mozilla.updater",
                 # 'xpcshell',
diff --git a/toolkit/components/telemetry/app/TelemetrySend.sys.mjs b/toolkit/components/telemetry/app/TelemetrySend.sys.mjs
index 81dd0bd..8c89ad6 100644
--- a/toolkit/components/telemetry/app/TelemetrySend.sys.mjs
+++ b/toolkit/components/telemetry/app/TelemetrySend.sys.mjs
@@ -1692,6 +1692,9 @@ export var TelemetrySendImpl = {
   },
 
   runPingSender(pings, observer) {
+    throw Components.Exception("", Cr.NS_ERROR_NOT_IMPLEMENTED);
+
+    /* eslint-disable-next-line no-unreachable */
     if (AppConstants.platform === "android") {
       throw Components.Exception("", Cr.NS_ERROR_NOT_IMPLEMENTED);
     }
diff --git a/toolkit/components/telemetry/moz.build b/toolkit/components/telemetry/moz.build
index 66b428d..2c4ab5c 100644
--- a/toolkit/components/telemetry/moz.build
+++ b/toolkit/components/telemetry/moz.build
@@ -8,10 +8,6 @@ include("/ipc/chromium/chromium-config.mozbuild")
 
 FINAL_LIBRARY = "xul"
 
-DIRS = [
-    "pingsender",
-]
-
 DEFINES["MOZ_APP_VERSION"] = '"%s"' % CONFIG["MOZ_APP_VERSION"]
 
 LOCAL_INCLUDES += [
