diff --git a/toolkit/themes/shared/desktop-jar.inc.mn b/toolkit/themes/shared/desktop-jar.inc.mn
index eafe624..84fec7a 100644
--- a/toolkit/themes/shared/desktop-jar.inc.mn
+++ b/toolkit/themes/shared/desktop-jar.inc.mn
@@ -141,6 +141,7 @@
   skin/classic/global/illustrations/shield-check.svg       (../../shared/illustrations/shield-check.svg)
   skin/classic/global/illustrations/shield-alert.svg       (../../shared/illustrations/shield-alert.svg)
   skin/classic/global/illustrations/no-connection.svg      (../../shared/illustrations/no-connection.svg)
+  skin/classic/global/illustrations/warning.svg            (../../shared/illustrations/warning.svg)
   skin/classic/global/illustrations/kit-concerned.svg      (../../shared/illustrations/kit-concerned.svg)
   skin/classic/global/illustrations/kit-happy.svg          (../../shared/illustrations/kit-happy.svg)
   skin/classic/global/media/picture-in-picture-open.svg    (../../shared/media/picture-in-picture-open.svg)
